README.md

```diff
@@ -83,7 +83,7 @@ For both unsupervised and supervised video editing, such as face-swap, please re
 
 ### Training
 
-**Note: It is important to use pytorch==1.0.0 for training. Higher versions of pytorch have strange bilinear warping behavior, because of it model diverge.**
+**Note: It is important to use pytorch==1.0.0 for training. Higher versions of pytorch have strange bilinear warping behaviour, because of its model diverge.**
```

Comment:
`behavior` vs `behaviour` is American vs British English, so not significant.

The sentence doesn't make sense regardless, it seems like it should be "and because of it the model diverges"